numba/types.py

```diff
@@ -1400,6 +1400,105 @@ class Slice3Type(Type):
     pass
 
 
+class ClassInstanceType(Type):
```

Comment:
Added in https://github.com/sklam/numba/commit/90f6a87c41fd22cc2c99b20cf7fa82ad6e55ce8b